nomenclature/testing.py

```diff
@@ -24,3 +46,12 @@ def assert_valid_yaml(path):
         raise AssertionError(
             "Parsing the yaml files failed. Please check the log for details."
         )
+
+
+def check_valid_structure(path: Path):
```

Comment:
For consistency with the previous function...
```suggestion
def assert_valid_structure(path: Path):
```